dev-libs/libaddressinput/libaddressinput-0~git.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="Google’s postal address library that powering Android and Chromium"
+DESCRIPTION="libaddressinput can provide information about \
+what input fields are required for a correct address input form for \
+any country in the world and can validate an address to highlight \
+input errors like missing required fields or invalid values."
+HOMEPAGE="https://github.com/googlei18n/libaddressinput"
+COPYRIGHT="2014 Google Inc.
+	2015 THL A29 Limited, a Tencent company
+	2015 Milo Yip
+	2006-2013 Alexander Chemeris
+	2002 JSON.org"
+LICENSE="Apache v2
+	JSON
+	MIT"
+REVISION="1"
+srcGitRev="a575c63de328d0c72237c55d1a05e191c245122a"
+SOURCE_URI="https://github.com/googlei18n/libaddressinput/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="f92b38a0291372aa97306626f2941485bd1c00bd3cc7e242a1cb9fcd50b62212"
+SOURCE_FILENAME="libaddressinput-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="libaddressinput-$srcGitRev"
+googletestVersion="1.8.0"
+SOURCE_URI_2="https://github.com/google/googletest/archive/release-$googletestVersion.tar.gz"
```

Comment:
You may want to try to decouple the need of gtest. I believe the library that the build system wants would be `libgtest_main`. Try to see if you could adapt the build system to use system gtest. It's OK if that's not possible.

Reply:
`pkgman search gtest` finds out a 'gtest_source' package. But cannot install it. It's weird because the pkgman reports ' "gtest_source": Name not found' rather than some more resonable errors...

Reply:
Same thing for me. And it seems it's not just gtest. I also have the "Name not found" error if I try to install e.g. hexedit_source.

Reply:
It's not possible due to [this][0] ticket.

Given that Arch Linux [install parts of gtest source][1] for the purpose of inclusion by other packages, the recipe for gtest could be modified to do the same. If you choose to do so, the folder to save source code should be `$dataDir/gtest/src` and it should resides in `_devel` package.

[0]: https://dev.haiku-os.org/ticket/9990
[1]: https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/gtest#n82